docs/http-clients.md

```diff
@@ -0,0 +1,17 @@
+# HTTP Clients
+
+To date, we have four different HTTP API clients:
+
+- [kubo-rpc-client](https://github.com/ipfs/js-kubo-rpc-client) - Official JS client for talking to Kubo over HTTP
```

Comment:
```suggestion
- [js-kubo-rpc-client](https://github.com/ipfs/js-kubo-rpc-client) - Official JS client for talking to Kubo over HTTP
```

Reply:
The npm repo is `kubo-rpc-client` (https://www.npmjs.com/package/kubo-rpc-client) so I was trying to keep with the naming convention that JS users would use. fine to change this

Reply:
Oh got it - make sense